app/assets/javascripts/ci_api/services/Transcript.js

```diff
@@ -21,6 +21,25 @@ export default class Transcript {
         this._appendMessage(msg, this.classes.Opener);
     }
 
+    addSkipToBottomLink(){
+
+       const chatContainer = document.getElementById("ciapiSkinChatTranscript")
+
+        if (chatContainer.scrollHeight > chatContainer.clientHeight) {
+            chatContainer.insertAdjacentHTML("beforeend", '<div id="skipToTop" class="skipToTopWithScroll"><a id="skipToTopLink" href="#" class="govuk-skip-link">Skip to top of conversation</a></div>');
+        }
+        else {
+            chatContainer.insertAdjacentHTML("beforeend", '<div id="skipToTop" class="skipToTopWithOutScroll"><a id="skipToTopLink" href="#" class="govuk-skip-link">Skip to top of conversation</a></div>');
```

Comment:
@nehakannaujia1 I feel there is a lot of duplication here. Could we create a function and maybe pass the class name in as it seems it's the only thing that is different?